### DISCUSSION TITLE
Diary-19-倒二？ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#337] [Announcements]
# Diary-19-倒二？ | Zi-Zi's Journey

2024.06.17-2024.06.30

https://gz-metal-cell.github.io/2024/06/27/Diary-19-%E5%80%92%E4%BA%8C%EF%BC%9F/

**Comment by promefire** [comment]
😭怎么这么多需要输入密码才能看的😭

**Reply by GZ-Metal-Cell** [reply]
😭里面写满了 pro 哥的坏话~
